modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -2161,6 +2169,9 @@ public static synchronized boolean isLocalHostChanged() throws IOException {
                 List<InetAddress> locAddrs = new ArrayList<>();
 
                 for (NetworkInterface itf : asIterable(NetworkInterface.getNetworkInterfaces())) {
+                    if (IGNORED_INTERFACE_NAMES.contains(itf.getName()))
+                        continue;
```

Comment:
Why we change something here (this method was used in TcpCommunicationSpi, TcpDiscoverySpi)?